### PR TITLE
Update UG and PPP

### LINKED ISSUE #174: [PE-D][Tester B] Invalid email address accepted for add command
Valid email address should be in the form of `xxxx@xxxx.xxxxx`

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: PeiYee88/ped#5

### PR DESCRIPTION
**Pull request by ChongCheeKaiClarence**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -434,46 +434,91 @@ Examples:
...
 
-* `TAG1` is the current name of the tag and `TAG2` is the new name of the tag.
+<div markdown="block" class="alert alert-info">
+**:information_source: Note:** 
 
-Example:
-* `tag edit t/friend t/bestFriend` changes the friend tag to a bestFriend tag.
+* The new Tag must not have the same name as any other existing tags.
+* `TAG1` represents the current name of the tag and `TAG2` represents the new name of the tag.
+
```
Comment (Ugholaf) [inline docs/UserGuide.md]: Might be good to let the users know that by editing the tag will change all the contacts and todos with `TAG1` to `TAG2`.

I am worried that they will think that this only affects one tag in one contact or something.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -434,46 +434,91 @@ Examples:
...
+Fantastic! You have successfully learnt how to change tags.
 
 <br>
 
 #### 7.4.4. Adding a Tag to a Contact: `tag add`
 
-Adds an existing tag to an existing contact.
+You can add a `TAG` from the tag list to a contact.
 * `Coming soon in v1.5`, we will upgrade `tag add` to add tags to todos.
```
Comment (Ugholaf) [inline docs/UserGuide.md]: Maybe can include how the user can add the tag to todos for now, so using the todo edit and key in all the existing tags plus new tags that the user wants to add.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -483,27 +528,47 @@ A contact can have any number of tags. Add as many as you want.
...
+
+Awesome! You have successfully learnt to add a tag to a contact.
 
 <br>
 
 #### 7.4.5. Removing a Tag from a Contact: `tag remove`
 
-Removes an existing tag from an existing contact.
+You can remove a `TAG` from a contact.
 * `Coming soon in v1.5`, we will upgrade `tag remove` to remove tags from todos.
```
Comment (Ugholaf) [inline docs/UserGuide.md]: Same as above, let the user knows how to remove tag from todos

**Review by Ugholaf — COMMENTED** [review]
LGTM, just some suggestions for the tag feature

File: docs/UserGuide.md [hunk 4]
```diff
@@ -434,46 +434,91 @@ Examples:
...
 
 Format: `tag delete t/TAG`
 
 <div markdown="block" class="alert alert-info">
 **:information_source: Note:** When `TAG` is deleted, `TAG` is removed from all the contacts which previously had it.
 </div>
 
-Example:
-* `tag delete t/family` deletes the `family` tag.
+**Expected Output in Command Box**
```
Comment (Eugene-Ong-W-X) [inline docs/UserGuide.md]:
```suggestion
**Expected Input in Command Box**
```

**Review by Eugene-Ong-W-X — COMMENTED** [review]
I was browsing your PR to check something, 1 typo caught my eye

File: docs/UserGuide.md (hunk 4 above)
Comment (ChongCheeKaiClarence) [reply docs/UserGuide.md]: Noted. Thanks!

File: docs/UserGuide.md (hunk 1 above)
Comment (ChongCheeKaiClarence) [reply docs/UserGuide.md]: I understand your concern. I'll make the changes.

File: docs/UserGuide.md (hunk 2 above)
Comment (ChongCheeKaiClarence) [reply docs/UserGuide.md]: I think i will link to your todo edit for more information

File: docs/UserGuide.md (hunk 3 above)
Comment (ChongCheeKaiClarence) [reply docs/UserGuide.md]: Solution is same as above

**Pull request merged**